### PR TITLE
Chore: remove GregorianCalendar usage

### PR DESCRIPTION
**Pull request by dbmalkovsky**
This is when creating a new OPD entry when `OPDEXTENDED=no` in the settings file

### CODE REVIEW

File: src/main/java/org/isf/opd/gui/OpdEdit.java [hunk 1]
```diff
@@ -508,8 +507,7 @@ private JButton getOkButton() {
 
 						try {
 							if (insert) {    // Insert
-								GregorianCalendar date = new GregorianCalendar();
-								opd.setProgYear(opdManager.getProgYear(date.get(Calendar.YEAR)) + 1);
+								opd.setProgYear(opdManager.getProgYear(LocalDate.now().getYear()) + 1);
```
Comment (mwithi) [inline src/main/java/org/isf/opd/gui/OpdEdit.java:510]: In the case OPDEXTENDED=yes we call .getProgYear() method passing the visitDate (so it calculates it within the specified year)

Comment (dbmalkovsky) [reply src/main/java/org/isf/opd/gui/OpdEdit.java:510]: Well that is a change from the behavior that I modified here isn't it?   It was using "NOW" and I just did the same.
This is a NEW  (ie, insert) so is there even a visit date?

Comment (mwithi) [reply src/main/java/org/isf/opd/gui/OpdEdit.java:510]: if you like you can blame the forgotten change among OP-719 (my bad)

Comment (dbmalkovsky) [reply src/main/java/org/isf/opd/gui/OpdEdit.java]: No need to find blame; I've updated the code; please review @mwithi

Comment (mwithi) [reply src/main/java/org/isf/opd/gui/OpdEdit.java]: Thanks!

**Pull request merged**